.github/workflows/build.yml

```diff
@@ -367,11 +367,12 @@ jobs:
         with:
           name: lib
           path: clients/cobol-dialect-api
-      - name: update version for PR
+      - name: update version with PR or build number
         working-directory: clients/cobol-lsp-vscode-extension
-        if: github.event_name == 'pull_request'
+        env:
+          BUILD_ID: ${{github.event_name == 'pull_request' && format('pr-{0}', github.event.pull_request.number) || github.run_number }}
```

Comment:
We need to be able to turn this off for builds that are to be published on the VSCode marketplace, as MS still does not support semver there.